main/store/state/index.js

```diff
@@ -174,6 +174,7 @@ const initial = {
     reveal: main('reveal', false),
     nonceAdjust: main('nonceAdjust', false),
     autohide: main('autohide', true),
+    exceptionReporting: main('exceptionReporting', true),
```

Comment:
could you nest this under another key such as `privacy`? this will leave the door open to a bit more organization if we want to add other privacy-related state in the future